lib/omnibus/publishers/artifactory_publisher.rb

```diff
@@ -23,6 +23,10 @@ def publish(&block)
       log.info(log_key) { 'Starting artifactory publisher' }
       safe_require('artifactory')
 
+      if packages.empty?
+        log.info(log_key) { 'No packages found, skipping publish' }
+      end
+
```

Comment:
We should probably put this message up in `Omnibus::Publisher#packages`:
https://github.com/chef/omnibus/blob/shain/add_publish_debug/lib/omnibus/publisher.rb#L68-L117

This will ensure all publishers benefit.